_docs/_latest/setup/install.md

```diff
@@ -60,7 +60,7 @@ steps below:
       # If the tag exists in the remote repository but you are unable to checkout the tag,
       # run command `git fetch --all` to fetch all the latest branch/tag information and run
       # the checkout command again.
-      git checkout tags/v2.0.0
+      git checkout tags/<tag_number>
```

Comment:
I think that should include the small `v` as that is part of the whole tag (v2.0.0, v2.1.0).  I am worried that people might miss that, when they paste in just the version number, i.e. 2.0.0.
```
git checkout tags/v<tag_number>
```
i.e. it will be
```
git checkout tags/v2.0.0
```